src/core/tests/visitors/op/glu.cpp

```diff
@@ -0,0 +1,73 @@
+// Copyright (C) 2018-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "ov_ops/glu.hpp"
+
+#include <gtest/gtest.h>
+
+#include "visitors/visitors.hpp"
+
+using ov::op::internal::GLU;
+using ov::op::v0::Parameter;
```

Comment:
```suggestion
using ov::op::internal::GLU, ov::op::v0::Parameter;
```
Now it is possible also.

Reply:
Possible, but separate `using` statements can be still preferred for clarity and maintainability.
Keeping as is to have consistency in the codebase.